manifests/plugin/container_gateway.pp

```diff
@@ -21,6 +23,7 @@
   Foreman_proxy::ListenOn $listen_on = 'https',
   Stdlib::HTTPUrl $pulp_endpoint = "https://${facts['networking']['fqdn']}",
   Stdlib::Absolutepath $sqlite_db_path = '/var/lib/foreman-proxy/smart_proxy_container_gateway.db',
+  Integer $sqlite_timeout = 30000,
```

Comment:
Should this be `Optional[Integer[1]]` and not force the 30000 as that's the default of the app anyway?

Also no idea what a sensible minimum setting is (the 1 above)

Reply:
Good idea, it should be optional.

Reply:
In the sqlite docs (https://www.sqlite.org/c3ref/busy_timeout.html) it says: "Calling this routine with an argument less than or equal to zero turns off all busy handlers." 

From that I think it would be fine to allow any integer.